pynest/nest/lib/hl_api_helper.py

```diff
@@ -564,7 +564,7 @@ def set_verbosity(level):
 
     # Defined in hl_api_helper to avoid circular inclusion problem with
     # hl_api_info.py
-    sr
+    sr("%s setverbosity" % level)
```

Comment:
`%` string substitution is deprecated in Py3. Rather use `'{} setverbosity'.format(level)`